feature-libs/my-account/organization/core/model/LoadStatus.ts

```diff
@@ -0,0 +1,28 @@
+export enum LoadStatus {
+  SUCCESS,
+  ERROR,
+}
+
+export interface ItemInfo<T> {
+  status: LoadStatus;
+  value: T;
+}
+
+//TODO: consider to make public LoaderState from core
+interface LoaderState<T> {
+  loading?: boolean;
+  error?: boolean;
+  success?: boolean;
+  value?: T;
+}
+
+export function mapToItemInfo<T>(currentState: LoaderState<T>): ItemInfo<T> {
```

Comment:
In the same util file for this method you can also crate util that takes the stream and does the `pairwise`, login check and then uses this function in map so it is not repeated in every service.